docs/tools/api_paths.yaml

```diff
@@ -0,0 +1,12 @@
+- name: notifier
```

Comment:
Not used anywhere?

Reply:
It was called in ``api_gen.py`` as ``EXTS``, which was defined in ``doc_generator.py`` as a path constant.